pkg/gather/clusterconfig/operators.go

```diff
@@ -314,6 +352,10 @@ func collectContainerLogs(ctx context.Context, coreClient corev1client.CoreV1Int
 	readCloser, err := req.Stream(ctx)
 	if err != nil {
 		klog.V(2).Infof("Failed to fetch log for %s pod in namespace %s for failing operator %s (previous: %v): %q", pod.Name, pod.Namespace, containerName, isPrevious, err)
+		// it is fine to not have a previous log
+		if isPrevious {
+			return nil
+		}
```

Comment:
I wouldn't do that. It can fail regardless `isPrevious` flag and it should always return the error in such case.